src/Mypage/LoginMypage.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './LoginMypage.scss';
+
+function LoginMypage() {
+  return (
+    <div className="bodyContainer">
+      <div className="sidebar">
+        <div className="sidebarNav">
+          <ul className="subMenu">
+            <Link className="textLink" to="">
+              <li className="sideMypage">
+                <div>마이페이지</div>
+              </li>
+            </Link>
```

Comment:
```suggestion
              <li className="sideMypage">
                <Link className="textLink" to="">마이페이지</Link>
              </li>
```
이렇게 활용해보세요!
ul태그 하위는 리스트 태그가 와야하고, 각각의 리스트들이 링크로서 존재하는 개념입니다.
불필요한 div는 삭제해주세요~